src/main/scala/errors/DateFormatError.scala

```diff
@@ -1,3 +1,3 @@
-package Errors
+package errors
```

Comment:
Again, needs to be *inside* the CSVValidation package to avoid polluting the global namespace.

Reply:
done